sympy/core/exprtools.py

```diff
@@ -1418,8 +1418,8 @@ def _pemexpand(expr):
                         ok = hit = True
                         l = b**e
                         il = b**-e
-                        for i, a in enumerate(args):
-                            args[i][1][0] = il*args[i][1][0]
+                        for j, a in enumerate(args):
```

Comment:
Since `a` is not being used (and is not being repeatedly referenced) how about,
```
for j in range(len(args)):
    args[j][1][0] = il*ags[j][1][0]
```

Reply:
Makes sense! (I just tried to get rid of the duplicated loop variable, didn't check the code that carefully.)